### PR TITLE
Add hotfixes for PvpTalentEntry and PvpTalentUnlockEntry

### PR DESCRIPTION
**Pull request by Golrag**
Last column for PvpTalentEntry is unknown for me :/

### CODE REVIEW

File: WowPacketParserModule.V7_0_3_22248/Hotfix/PvpTalentEntry.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    {
+        public uint SpellID { get; set; }
+        public uint OverridesSpellID { get; set; }
+        public string Description { get; set; }
+        public uint TierID { get; set; }
+        public uint ColumnIndex { get; set; }
+        public uint Flags { get; set; }
+        public uint ClassID { get; set; }
+        public uint SpecID { get; set; }
+        public uint Unknown { get; set; }
```
Comment (Shauren) [inline WowPacketParserModule.V7_0_3_22248/Hotfix/PvpTalentEntry.cs]: Role. (ChrSpecializationEntry::Role)

File: WowPacketParserModule.V7_0_3_22248/Hotfix/PvpTalentEntry.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+{
+    [HotfixStructure(DB2Hash.PvpTalent, HasIndexInData = false)]
+    public class PvpTalentEntry
+    {
+        public uint SpellID { get; set; }
+        public uint OverridesSpellID { get; set; }
+        public string Description { get; set; }
+        public uint TierID { get; set; }
+        public uint ColumnIndex { get; set; }
+        public uint Flags { get; set; }
```
Comment (Shauren) [inline WowPacketParserModule.V7_0_3_22248/Hotfix/PvpTalentEntry.cs]: How did you determine this as flags on a column that is always 0 and is not referenced by client?

File: WowPacketParserModule.V7_0_3_22248/Hotfix/PvpTalentEntry.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace WowPacketParserModule.V7_0_3_22248.Hotfix
+{
+    [HotfixStructure(DB2Hash.PvpTalent, HasIndexInData = false)]
+    public class PvpTalentEntry
+    {
+        public uint SpellID { get; set; }
+        public uint OverridesSpellID { get; set; }
+        public string Description { get; set; }
+        public uint TierID { get; set; }
```
Comment (Shauren) [inline WowPacketParserModule.V7_0_3_22248/Hotfix/PvpTalentEntry.cs]: starting from this field all of them are signed ints

File: WowPacketParserModule.V7_0_3_22248/Hotfix/PvpTalentEntry.cs (hunk 2 above)
Comment (Golrag) [inline WowPacketParserModule.V7_0_3_22248/Hotfix/PvpTalentEntry.cs]: Woops, this is a guessed name. Nothing more than that I am afraid. I should have mentioned it. Reason why I guessed it was due to the similarities with Talent.db2.

**Pull request merged**